ghmoon

```diff
@@ -218,8 +218,9 @@ class GHRepo:
         self.repo, self.config = repo, config
         self.path = config.get("worktree", os.path.expanduser(f"~/.ghmoon/{repo}"))
         os.makedirs(self.path, exist_ok=True)
-        self.git = Git(f"{self.path}/git", f"https://github.com/{self.repo}.git") # TODO pass token to private repos
-        self.gh = GH(repo=self.repo, path=f"{self.path}/git", token=config.get("token"))
+        token=config.get("token")
```

Comment:
```suggestion
        token=config.get("token", "")
```
Without the empty string as the default, won't you get a `"None"` in the generated URL below?

Reply:
Can you even run ghmoon without the token? That was my impression.

Reply:
I just noticed you need a bigger change if token is None shoud be allowed:

https://token**@**github.com 
https://github.com

Typ så här:
str=f"{token}@" if token else: ""